en/identity-server/next/docs/guides/user-self-service/user-username-recovery.md

```diff
@@ -0,0 +1,40 @@
+# Username recovery for users
+
+Users may recover a forgotten username from an application sign-in page by following the steps below.
+
+!!! note "Before you begin"
+
+    Administrators should [enable username recovery]({{base_path}}/guides/account-configurations/account-recovery/username-recovery/) for users of the organization.
+
+## Recover a forgotten username
+
+To reset the username, the user should follow these steps.
```

Comment:
reset -> recover?

Reply:
Fixed by https://github.com/wso2/docs-is/pull/4858/commits/7c797952be0d1b954991ed2889ed15992b0564b4